containers/resalloc.Containerfile

```diff
@@ -0,0 +1,30 @@
+# FROM registry.access.redhat.com/ubi9/ubi
+# FROM quay.io/centos/centos:stream9
+FROM quay.io/centos/centos:stream8
+USER 0
+
+# RUN dnf install -y dnf-plugins-core https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+RUN dnf install -y dnf-plugins-core https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+# RUN dnf config-manager --set-enabled powertools
+
+# Deployment instructions are described here
+# https://github.com/praiskup/resalloc/blob/master/docs/start-resalloc-server.txt
+
+RUN dnf install -y ansible \
```

Comment:
With stream9 you need `--enablerepo crb`  https://bugzilla.redhat.com/show_bug.cgi?id=2268623